yarn.lock

```diff
@@ -2017,24 +1724,16 @@
   integrity sha512-NuHqBY1PB/D8xU6s/thBgOAiAP7HOYDQ32+BFZILJ8ivkUkAHQnWfn6WhL79Owj1qmUnoN/YPhktdIoucipkAQ==
 
 "@yaireo/tagify@^4.9.8":
-  version "4.9.8"
-  resolved "https://registry.yarnpkg.com/@yaireo/tagify/-/tagify-4.9.8.tgz#67082ecca81fbbec4d75bb95581280f27af22fba"
-  integrity sha512-JqRTN4vb5tx06PJIRIOAlriO/B6u+a3jD/VBb5x6E1MOlC4wrMfaLHmNnrMpiZCHMXox7pVmrtNuyBe4YReJ4w==
+  version "4.17.6"
```

Comment:
Prolly good to read their release notes for the versions between 4.9.8 and 4.17.6 to see if we have to change anything in our code (or at least manually test the tagify functionality) (or revert this hunk of the diff?)